test/gurka/test_traffic.cc

```diff
@@ -14,129 +15,6 @@
 
 using namespace valhalla;
 
-/** Copy of raw header for use with sizeof() **/
-typedef struct {
-  char name[100];
-  char mode[8];
-  char owner[8];
-  char group[8];
-  char size[12];
-  char mtime[12];
-  char checksum[8];
-  char type;
-  char linkname[100];
-  char _padding[255];
-} mtar_raw_header_t_;
-
-/*************************************************************/
-
-// Helper function for updating the BD edge in the following test
-void update_all_edges_but_bd(const valhalla::gurka::map& map,
```

Comment:
Moving a more general implementation of this into the new `test/util/traffic_utils.h`